en/06_Project_Governance/06_Minor_release_policy.md

```diff
@@ -0,0 +1,100 @@
+
+### Pre-release
+
+Silverstripe CMS does not usually tag alpha releases for minor releases.
+
+Approximately 6 weeks prior to the anticipated stable minor release, a beta minor release is published. Once a beta release is tagged, any new feature or API that didn't make it to the beta should be targeted to the follow up minor release. This allows the CMS development team to perform a regression test on the beta release with confidence that no additional regressions will be introduced before the stable release.
+
+
+Approximately 2 weeks prior to the anticipated stable minor release, a release candidate is tagged. Once a release candidate is tagged, only critical impact bug fixes can be added to the release. The release candidate is sent to an external auditor for a security review. Any security patches which will be included in the stable release are also sent to the auditor.
+
+### Stable release
+
+New minor releases of Silverstripe CMS are tagged around April and October each year.
+
+## Minor release support timeline
+
+The minor release support timeline follows similar phases to those of a major release, but are more condensed.
+
+### Bug and security fixes
+
+A Silverstripe CMS minor release line enters the *bug and security fixes* phase once it is tagged *stable*.
+
+A minor release in the *bug and security fixes* phase receives:
+- regular patches for bugs at all impact levels
+- security patches for all vulnerabilities.
+
+It does **not** receive:
+- new feature
+- new APIs.
+
+A minor release line stays in the *bug and security fixes* phase until a follow up minor release is tagged.
+
+### Security fixes only
+
+Once the follow up minor release is tagged, the minor release that was in the *bug and security fixes* phase transition to a *security fixes only* phase.
+
+In the *security fixes only* phase, a Silverstripe CMS minor release line only receives patches for high impact security issues defined has having a CVSS of 7.0 or above.
+
+
+### End-of-life
+
+Six months after entering the *security fixes only* phase, the minor release goes *end-of-life*. No more security patches will be shipped for that minor release.
+
+### The last minor release in a major release line
+
+Shortly before tagging a new major release, one last minor release will be tagged for the major release line in active development.
+
+The last minor release in a major release line will have an extended support phases in line with our major release policy:
+- the last minor release stays in the *bug and security fixes* phase for one year
+- then it enters the *security fixes only* phase for another year
+- then the minor release goes end of life along with its corresponding major release line.
+
+## Security patch windows
+
+Silverstripe CMS aims to deliver security patches at pre-defined intervals to minimise upgrade woes for Silverstripe CMS projects.
```

Comment:
```suggestion
Silverstripe CMS aims to deliver security patches at pre-defined intervals.
```

Reply:
You don't deliver security patches to make it easier to upgrade, you deliver them to deliver security patches :-)

Reply:
I read this as meaning "we have a specific cadence of releasing patches at pre-defined intervals instead of just releasing these ad-hoc to minimise upgrade woes for Silverstripe CMS projects."

Reply:
OK maybe change "to minimise upgrade woes for Silverstripe CMS projects" to "to make it easier to plan upgrading Silverstripe CMS projects so they can receive the security patches"

Reply:
Went with:
> Silverstripe CMS tags patch releases of individual modules whenever relevant bug fixes or other very low risk changes are merged into a supported patch release branch.